Documentation on System PCG for Self Hosted Palette Installation (#1348)
spectrocloud/librarium: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

Reviewer: @rahulhazra97 After our discussion about where users choose this option, we don't need a separate how-to section.

Reviewer: @rahulhazra97 Resources would be better listed as follows - I understand this isn't part of your update project: 

**Resources**

- [Install MAAS](https://maas.io/)

- [MAAS Fresh Install](https://maas.io/docs/how-to-install-maas)

- [Manage MAAS User Accounts](https://maas.io/docs/how-to-manage-user-accounts#heading--api-key)

Author: Thanks, Rita. Yeah, I fixed it.

Reviewer: @rahulhazra97 The first diagram in this document needs to be called out somehow. Something like:

"As the diagram shows, Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation, or jump box."

> # Overview 

Author: Mentioned in architecture.

> # Overview 

Reviewer: @rahulhazra97 I thought this needed some clarification in the architecture section and offered a revision of the third bullet item.

# Overview 

> The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. The primary function of the PCG is to facilitate connectivity between Palette and a private cloud behind a NAT gateway or firewall. The PCG traverses any NAT gateways or firewalls to establish a permanent connection with Palette. In this way, the function of the PCG is similar to that of a reverse proxy.

Reviewer: ```suggestion
The Private Cloud Gateway (PCG) supports private cloud and data center environments. Its function is similar to that of a reverse proxy. The PCG facilitates connectivity between Palette and a private cloud that exists behind a NAT gateway or firewall. The PCG traverses any NAT gateways or firewalls to establish a permanent connection with Palette.
```

> PCG is a Kubernetes cluster responsible for supporting Palette in a private network environment. All host clusters deployed through Palette will communicate with PCG.

Reviewer: @rahulhazra97 I noticed this line needed some work.
